### DISCUSSION TITLE
Is it possible to limit the max-charge capacity beyond which the PC stops charging?

### DISCUSSION THREAD

**Discussion by ghost** [AdnanHodzic/auto-cpufreq#492] [Q&A]
I am asking if it's possible to set a max charge limit of 60% so ones it's there, my PC just stops charging unless it hits something like 40%.

Basically I want to keep my battery in the range of 40% to 60% and I found out that TLP supports it, although only for thinkpad laptops.

**Comment by SunkenHero** [comment]
To my knowledge auto-cpufreq does not support charge thresholds. I read a little bit on the TLP documentation and got to [this](https://linrunner.de/tlp/settings/bc-vendors.html#supported-hardware). There are some more supported models than thinkpads only or am i doing something wrong?
